sys/riscv/riscv/pmap.c

```diff
@@ -935,8 +935,13 @@ pmap_bootstrap(vm_paddr_t kernstart, vm_size_t kernlen)
 	 */
```

Comment:
A few notes on the commit log:

- I would just use 'riscv' as the prefix.  We don't generally use 'cheri-riscv' prefixes.
- While we don't use punctuation, we do tend to capitalize the first word in the title, and we aim to fit the title in the recommended 72 columns, so maybe:

```
riscv: Ensure bounds for KVA map cover all of kmem
```

- For the body, just be sure to wrap the commit log (in emacs I use M-q to auto-wrap commit log paragraphs).